<?xml version="1.0"?>
<launch>

  <param name="use_sim_time" value="true"/>

  <!-- Visualization -->
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find amcl3d)/rviz/amcl3d.rviz"/>
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_velodyne" args="0 0 0 0 0 0     /base_link    /velodyne   10" />
  <include file="$(find amcl3d)/launch/amcl3d.launch">
    <arg name="map_name_path"   value="$(find amcl3d)/data/"/>
    <arg name="init_x"          value="0"/>
    <arg name="init_y"          value="0"/>
    <arg name="init_z"          value="0"/>
    <arg name="init_a"          value="0"/>
    <arg name="num_particles"   value="600"/>
    <arg name="alpha"           value="0.5"/>
    <arg name="take_off_height" value="-0.10"/>
  </include>

</launch>
